x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/custom_assets_extension.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { type CustomAssetsAccordionProps, CustomAssetsAccordion } from '@kbn/fleet-plugin/public';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '../../common/hooks/use_kibana';
+import { cloudPosturePages } from '../../common/navigation/constants';
+
+const SECURITY_APP_NAME = 'securitySolutionUI';
```

Comment:
we should probably get this from the security solution context

Reply:
worth a TODO maybe